### PR TITLE
build: move to node v12.18.3

### LINKED ISSUE #660: [RFC] Base Node version update
Currently our repo is built around `v10.18.1` (`.nvmrc`, Circle CI, etc), however the current LTS version of Node is `v12.16.3`, is there reason _not_ to update?

### PR DESCRIPTION
**Pull request by Westbrook**
## Description
FYI: to leverage this branch you MUST update to Node 12.18.3, [nvm](https://github.com/nvm-sh/nvm) is great for this.

Upgrade Node.js usage to v12+.

Side benefit, early returns are seeing small performance win in our CI cycles with this update!

## Related Issue
fixes #660 

## Motivation and Context
Node current is v14, LTS is v12, in v12/13 there are important changes to the use of `type="module"` that will prevent users from being able to actively work with our repo if they are these versions. Let's move up!

## How Has This Been Tested?
- all unit and regression testing passes

## Types of changes
- [x] chore

## Checklist:
- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: __snapshots__/Search.md [hunk 1]
```diff
@@ -2,6 +2,32 @@
...
+    id="button"
+    label="Reset"
+    tabindex="-1"
+    variant=""
+  >
+  </sp-clear-button>
+</form>
+```
+
+#### `dispatches events when using the "clear" button`
```
Comment (adixon-adobe) [inline __snapshots__/Search.md:29]: Should this be checked in?

File: packages/button/test/button.test.ts [hunk 2]
```diff
@@ -10,10 +10,9 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import '@spectrum-web-components/button/sp-button.js';
-import { Button } from '@spectrum-web-components/button';
-import { html } from 'lit-element';
-import { fixture, elementUpdated, expect } from '@open-wc/testing';
+import '../sp-button.js';
+import { Button } from '..';
+import { html, fixture, elementUpdated, expect } from '@open-wc/testing';
```
Comment (adixon-adobe) [inline packages/button/test/button.test.ts:15]: Was this `html` change just something you noticed while fixing the other imports?

File: packages/checkbox/test/checkbox.test.ts [hunk 3]
```diff
@@ -178,6 +178,7 @@ describe('Checkbox', () => {
         HTMLElement.prototype.focus.apply(el);
         el.focusElement.dispatchEvent(new CustomEvent('focusout'));
         await nextFrame();
+        await nextFrame();
```
Comment (adixon-adobe) [inline packages/checkbox/test/checkbox.test.ts:181]: This seems suspect

**Comment by Westbrook** [comment]
@adixon-adobe pulled out all those Safari changes, should be clean as a whistle!

**Pull request merged**